gradle/libs.versions.toml

```diff
@@ -38,15 +39,16 @@ ktfmt = "0.44"
 # replaces nuValidator's log4j dependency
 # something related to log4j-1.2-api is apparently broken in 2.17.2
 log4j = "2.17.1"
-msgpack = "0.9.0"
-nexusPublishPlugin = "1.3.0"
+msgpack = "0.+"
```

Comment:
Not _entirely_ comfortable with a `0.+`, given the instability of `0` major. @bioball / @stackoverflow; thoughts?

Reply:
I believe @holzensp is saying that we should declare this as `0.9.+`, rather than `0.+`, because per semver rules, new `0.x` versions can have breaking changes.

Reply:
The way I read the semver spec, all bets are off for 0.x.y versions, and 0.9.8 -> 0.9.9 may break everything.
0.9.+ risks missing that 0.10 exists. If you prefer it nevertheless, I don’t mind making this change.

Reply:
Ah, you're right. And, looking at their [release history](https://github.com/msgpack/msgpack-java/releases), they have breaking changes in patch versions.

Maybe it's best to fully specify the version here; `msgpack = "0.9.8"`.

Reply:
Personally I'm more worried about falling way behind the latest version than about the build failing to catch an incompatible change. But to move this forward, I'll change back to `msgpack = "0.9.8"`.

By the way, I wasn't happy with the state of msgpack's Java library.
(They also didn't reply to the issues and PRs I opened.)
Eventually I wrote a replacement that I'll soon turn into a public GitHub project. From the README:

## MiniPack at a Glance

* Complete implementation of the MessagePack binary serialization format.
* Clean and modern API with [JSpecify](https://github.com/jspecify/jspecify) nullness annotations.
* Designed to be correct, efficient, and extensible.
* Embraces Java NIO: Heap and direct byte buffers, channels, channel transfers, gathering writes, buffer pooling.
* Fast, correct, and customizable string encoding/decoding with java.nio.charset.CharsetEncoder/CharsetDecoder.
* Optimized handling of strings used as identifiers.
* Small JAR size (~50 KB).
* No use of reflection or internal/unsafe JDK classes.
* Ships as Java (JPMS) module.
* No dependencies other than [JSpecify](https://central.sonatype.com/artifact/org.jspecify/jspecify) (~3 KB).
* Compatible with [GraalVM Native Image](https://www.graalvm.org/latest/reference-manual/native-image/).
* Thoroughly benchmarked with [JMH](https://github.com/openjdk/jmh).
* Thoroughly tested with [jqwik](https://jqwik.net/).

Reply:
Sounds good! Happy to entertain switching from msgpack-java to minipack.

Reply:
@bioball ready to squash and merge from my side.